torch/_meta_registrations.py

```diff
@@ -673,6 +673,70 @@ def linalg_lu_factor_ex_meta(
     return LU, pivots, info
 
 
+@register_meta([aten.linalg_lu_solve.default, aten.linalg_lu_solve.out])
+def linalg_lu_solve_meta(
+    LU: Tensor,
+    pivots: Tensor,
+    B: Tensor,
+    *,
+    left: bool = True,
+    adjoint: bool = False,
+    out: Tensor = None,
```

Comment:
```suggestion
    out: Optional[Tensor] = None,
```

Reply:
No longer needed. I've changed it back to use out_wrapper.